sloglint.go

```diff
@@ -438,3 +454,26 @@ func argsOnSameLine(fset *token.FileSet, call ast.Expr, keys, attrs []ast.Expr)
 
 	return false
 }
+
+func forbiddenKeysUsed(forbiddenKeys []string) func(string) bool {
+	return func(s string) bool {
+		// Strip quotes if any.
+		unquoted := trimQuotes(s)
+		return slices.Contains(forbiddenKeys, unquoted)
+	}
+}
+
+func valueChanged(handler func(string) string) func(string) bool {
+	return func(s string) bool {
+		return handler(s) != s
+	}
+}
+
+func trimQuotes(s string) string {
```

Comment:
Sorry, why is it needed? I've never seen the use of quoted keys in logger calls.

Reply:
`getKeyName` returns the `lit.Value` when `lit.Kind=token.String`.

The issue is that `lit.Value` is the literal representation of the string and not the string value. This means, that the `lit.Value` contains the quotes used to define the string. 
This was never an issue because the case converters ignore other chars like `, " and '

Reply:
TIL :) Should we trim quotes before returning from  `getKeyName` then?

Reply:
done